content/post/2018-04-02-sct-encoding.markdown

```diff
@@ -0,0 +1,439 @@
+---
+author: Jacob Hoffman-Andrews, Boulder developer
+date: 2018-04-02T00:00:00Z
+excerpt: How Signed Certificate Timestamps get embedded in certificates
+title: "Engineering deep dive: Encoding of SCTs in certificates"
+slug: sct-encoding
+---
+
+Let's Encrypt recently [launched SCT embedding in
+certificates](https://community.letsencrypt.org/t/signed-certificate-timestamps-embedded-in-certificates/57187).
+This feature allows browsers to check that a certificate was submitted to a
+Certificate Transparency log. As part of the launch, we did a thorough review
```

Comment:
It might be worth a hyperlink to https://en.wikipedia.org/wiki/Certificate_Transparency or https://www.certificate-transparency.org/ to provide a jump-off point for folks completely unfamiliar with CT.